shardingsphere-db-protocol/shardingsphere-db-protocol-core/src/main/java/org/apache/shardingsphere/db/protocol/error/CommonErrorCode.java

```diff
@@ -37,6 +37,12 @@
     
     RESOURCE_NOT_EXIST(11004, "C11004", "Can not drop resources, resources %s do not exist."),
     
+    REPLICA_QUERY_RULE_NOT_EXIST(11005, "C11005", "Replica query rule do not exist."),
+    
+    REPLICA_QUERY_RULE_DATASOURCE_NOT_EXIST(11006, "C11006", "Datasource %s in Replica query rule do not exist."),
```

Comment:
`Replica` should be lower case if not in the first word of sentence.

Reply:
ok